newm-server/src/main/kotlin/io/projectnewm/server/features/song/repo/SongRepository.kt

```diff
@@ -1,7 +1,12 @@
 package io.projectnewm.server.features.song.repo
 
 import io.projectnewm.server.features.song.model.Song
+import java.util.UUID
 
 interface SongRepository {
-    suspend fun getSongs(): List<Song>
+    suspend fun add(song: Song, ownerId: UUID)
+    suspend fun update(song: Song, songId: UUID, requesterId: UUID)
+    suspend fun delete(songId: UUID, requesterId: UUID)
+    suspend fun get(songId: UUID): Song
+    suspend fun getAll(ownerId: UUID): List<Song>
```

Comment:
nit: maybe this can get `getAllByOwnerId`? I feel like we'll eventually want to get lists of songs using different values/ids

Reply:
Good call. Done!